src/components/ChatEntry.js

```diff
@@ -1,22 +1,67 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import { DateTime } from 'luxon';
 
 const ChatEntry = (props) => {
-  return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
-      <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
-      </section>
-    </div>
-  );
+  const id = props.id;
+  const sender = props.sender;
+  const body = props.body;
+  const timeStamp = props.timeStamp;
+  const liked = props.liked;
+  const updateLikes = props.updateLikes;
```

Comment:
Another way to directly access these props values could be to deconstruct props inside L6 instead of initializing new values in L7-L12, like so:
```
const ChatEntry = ({id, sender, body, timeStamp, liked updateLikes}) => {
```